examples/run/dp1.x-lammps-ABACUS-lcao-dpks/fcc-al/machine.json

```diff
@@ -0,0 +1,106 @@
+{
+  "api_version": "1.0",
+  "deepmd_version": "2.0.1",
```

Comment:
It's unclear why this file is put under a directory named `dp1.x`...

Reply:
I see. I will move it to dp2.x...